lms/djangoapps/discussion/templates/discussion/discussion_mfe_embed.html

```diff
@@ -1,41 +0,0 @@
-## mako
-
-<%namespace name='static' file='../static_content.html'/>
-
-<%page expression_filter="h"/>
-
-<%!
-import json
-from django.utils.translation import ugettext as _
-from openedx.core.djangolib.js_utils import js_escaped_string
-%>
-
-<%block name="js_extra">
-<script>
-    window.addEventListener("load", function () {
-        const mfeUrl = (new URL("${discussions_mfe_url  | n, js_escaped_string}"));
-        const iframe = document.querySelector('#discussions-mfe-tab-embed');
-        let timestamp = "?no-cache=" + Date.now();
-        let showInIframe = "&inIframe=" + "${show_in_iframe | n, js_escaped_string}";
-        iframe.src = mfeUrl.href + window.location.hash.slice(1) + timestamp + showInIframe;
-        window.addEventListener("message", (event) => {
-            const payload = event.data.payload;
-            if (event.origin !== mfeUrl.origin) {
-                return;
-            }
-            if (event.data.type === "discussions.navigate" && payload.path){
-                window.history.pushState(null, "", "#" + payload.path);
-            }
-            if (event.data.type === "plugin.resize") {
-                iframe.height = payload.height;
-            }
-        });
-    });
-</script>
-</%block>
-
-<section class="discussion discussion-board page-content-container" id="discussion-container"
-         data-course-id="${course_key}">
-    <%include file="_switch_experience_fragment.html" />
```

Comment:
@muhammadadeeltajamul can we remove this html file as well. I dont think we need this now!

Reply:
@saadyousafarbi "_switch_experience_fragment.html" file is also being used in legacy experience.